### PR TITLE
WILL-1783/Adding video url to composite image widget.

### PR DESCRIPTION
**Pull request by SorenThorup**

### CODE REVIEW

File: src/Models/ACF/Composite/CompositeFieldGroup.php [hunk 1]
```diff
@@ -1015,13 +1023,35 @@ private static function registerHooks()
...
 
             return $valid;
         }, 10, 4);
+        add_filter(sprintf('acf/validate_value/key=%s', self::VIDEO_LINK_FIELD), function ($valid, $value) {
+            if( $valid !== true ) {
+                return $valid;
+            }
+            if (strpos($value, 'vimeo') === false) {
+                $valid = 'Url must be a Vimeo url';
+                //$valid = false;
```
Comment (MFlor) [inline src/Models/ACF/Composite/CompositeFieldGroup.php:1039]: Slettes?

File: src/Models/ACF/Composite/CompositeFieldGroup.php [hunk 2]
```diff
@@ -1015,13 +1023,35 @@ private static function registerHooks()
...
+            }
+            return $valid;
+        }, 10, 4);
     }
+
+    /*public static function validateVideoUrl($valid, $value, $field, $input_name) {
```
Comment (MFlor) [inline src/Models/ACF/Composite/CompositeFieldGroup.php:1045]: Slettes?

File: src/Models/ACF/Composite/CompositeFieldGroup.php (hunk 1 above)
Comment (SorenThorup) [reply src/Models/ACF/Composite/CompositeFieldGroup.php]: fixed.

File: src/Models/ACF/Composite/CompositeFieldGroup.php (hunk 2 above)
Comment (SorenThorup) [reply src/Models/ACF/Composite/CompositeFieldGroup.php]: But ofc...

**Pull request merged**